tensorflow/contrib/tensorrt/convert/trt_optimization_pass.cc

```diff
@@ -232,8 +232,18 @@ tensorflow::Status TRTOptimizationPass::Optimize(
   tensorflow::grappler::GraphProperties static_graph_properties(item);
   TF_RETURN_IF_ERROR(static_graph_properties.InferStatically(true));
   tensorflow::tensorrt::convert::ConversionParams cp;
+
+  std::vector<string> nodes_to_preserve;
+  for (const auto& n : item.NodesToPreserve()) {
+    auto tokens = str_util::Split(n, ":");
+    string s = tokens.at(0);
```

Comment:
As chatted offline please remove the following for loop since colon is not a valid character for node names and thus there must be at most one colon in each token.

Reply:
Done